appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/JdbcContractTestBase.java

```diff
@@ -17,7 +17,6 @@
 
 import static io.opentelemetry.proto.trace.v1.Span.SpanKind.SPAN_KIND_CLIENT;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertThat;
```

Comment:
Interesting, I don't really know how that worked before...

Reply:
was added on this commit 
https://github.com/aws-observability/aws-otel-java-instrumentation/commit/c8db84a5af5647be351a2477351f51d8f12e47b4

I'm not sure how did it work 🤔

my guess is compiler could see the usage and based on that could decide which static import to be used, but regardless I think the way to go is to remove the junit one, as not needed and not used in the whole project 👀